src/components/interfaces.ts

```diff
@@ -19,11 +22,20 @@ export interface ComponentProperty {
   inlineType?: TypeDefinition;
   defaultValue?: string;
   analyticsTag?: string;
+  deprecatedTag?: string;
+  visualRefreshTag?: string;
+  i18nTag?: true | undefined;
+  systemTags?: Array<string>;
 }
 
 export interface ComponentRegion {
   name: string;
   description?: string;
+  displayName?: string;
+  isDefault: boolean;
+  deprecatedTag?: string;
+  visualRefreshTag?: string;
+  i18nTag?: true | undefined;
```

Comment:
These fields were always emitted in the docs, but because of loose type validation, it worked even without declarations here.